src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -351,7 +351,7 @@ func (c controller) postInstallConfigs() error {
 	c.log.Infof("Waiting for cluster version operator: %t", c.WaitForClusterVersion)
 
 	if c.WaitForClusterVersion {
-		err = c.waitingForClusterVersion()
+		err = c.waitingForClusterVersion(context.TODO())
```

Comment:
This must be the context of postInstallConfigs so it can be cancelled. I suggest you wait for @tsorya fix to be merged first because interfaces has changed here